src/deploy/functions/prepare.ts

```diff
@@ -422,3 +417,46 @@ export function resolveCpuAndConcurrency(want: backend.Backend): void {
     }
   }
 }
+
+/**
+ * Exported for testing purposes only.
```

Comment:
This is misleading. "for testing purposes only" very much means, in very other context: "I need to export this to write tests against it", *not* "this is exposed for an `internaltesting` command". Please clarify that it's for the command, rather than simply "testing".